docs/section21/Rule21-19.md

```diff
@@ -0,0 +1,44 @@
+
+# Boiler - Rule 21-19  
+
+**Rule ID:** 21-19  
+**Rule Description:** All boilers in the baseline building design shall be modeled at the minimum efficiency levels, both part load and full load, in accordance with Tables G3.5.6.  
+**Rule Assertion:** B-RMR  
+**Appendix G Section:** Section 21 Boiler  
+**Appendix G Section Reference:** Section G3.1.2.1 General Baseline HVAC System Requirements - Equipment Efficiencies
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-1, 5, 7, 11.2, 12, 1a, 7a, 11.2a, 12a.
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:**  
+
+1. get_baseline_system_types()
+
+**Applicability Checks:**  
+
+- Get B-RMR system types: `baseline_hvac_system_dict = get_baseline_system_types(B-RMR)`
+
+  - Check if B-RMR is modeled with at least one air-side system that is Type-1, 5, 7, 11.2, 12, 1a, 7a, 11.2a, 12a, continue to rule logic: `if any(sys_type in baseline_hvac_system_dict.keys() for sys_type in ["SYS-1", "SYS-5", "SYS-7", "SYS-11.2", "SYS-12", "SYS-1A", "SYS-7A", "SYS-11.2A", "SYS-12A"]): CHECK_RULE_LOGIC`
+
+  - Else, rule is not applicable to B-RMR: `else: RULE_NOT_APPLICABLE`
+
+## Rule Logic:  
+
+- For each boiler in B-RMR: `for boiler_b in B_RMR.RulesetModelInstance.boilers:`
+
+  **Rule Assertion - Component:**
+
+  - Case 1: For each boiler, if its rated capacity is less than 300,000Btuh, and its efficiency metric is AFUE, and its efficiency is 80%: `if ( boiler_b.rated_capacity < 300000 ) AND ( boiler_b.efficiency_metric == "ANNUAL_FUEL_UTILIZATION" ) AND ( boiler_b.efficiency == 0.8 ): PASS`
+
```

Comment:
@mkarpman do we need to convert the minimum eff in a specific metric to other metrics and verify the value? For example, E+ only takes in Et, the RMR is likely to include Et on all 3 cases. 
The challenge with this is using standard conversion methodology, one option would be to follow the PRM RM and document the same out here.

Reply:
as discussed on the RDS call, 
![image](https://user-images.githubusercontent.com/8323384/155231081-432282ef-dee3-44ed-9938-ed7ce5871d78.png)